Makefile.in

```diff
@@ -59,7 +59,7 @@ DOCFILES = README.md config LICENSE CHANGELOG.md docs/Docker.md docs/INSTALL.md
 
 ifneq ("$(wildcard /etc/redhat-release)","")
 RHEL = $(shell cat /etc/redhat-release | grep -E "(Red Hat Enterprise Linux Server|CentOS)" | wc -l)
-RHEL_VERSION = $(shell rpm --eval "%{centos_ver}")
+RHEL_VERSION = $(shell rpm --eval "%{rhel}")
```

Comment:
Does this return also the correct information on CentOS?

Reply:
yep:
**centos 6**
```
[alex@centos6minimal ~]$ cat /etc/rpm/macros.dist 
# dist macros.

%rhel 6
%centos 6
%centos_ver 6
%dist .el6
%el6 1
```

**centos 7**
```
[alex@centos7full ~]$ cat /etc/rpm/macros.dist 
# dist macros.

%centos_ver 7
%centos 7
%rhel 7
%dist .el7
%el7 1
```

RHEL uses `%rhel` which would have broken if using `centos_ver`